memdown.js

```diff
@@ -128,6 +128,16 @@ function MemDOWN (location) {
   this._store[this._location] = this._store[this._location] || createRBT()
 }
 
+MemDOWN.clearGlobalStore = function (strict) {
```

Comment:
non-strict simply replaces the globalStore with an empty object, strict deletes all the keys inside of it.  The difference has to do with in use instances keeping a reference to the global store which would would not be affected by simply replacing it.